CONTRIBUTING.md

```diff
@@ -1,17 +1,23 @@
 # Contributing
 
-Before you begin, read through some of the Terraform documentation on [Extending
+Before you begin, read through the Terraform documentation on [Extending
 Terraform](https://www.terraform.io/docs/extend/index.html) and [Writing Custom
 Providers](https://learn.hashicorp.com/collections/terraform/providers).
 
-Finally the [HashiCorp Provider Design
+Finally, the [HashiCorp Provider Design
 Principles](https://www.terraform.io/docs/extend/hashicorp-provider-design-principles.html)
 explore the underlying principles for the design choices of this provider.
 
+## Prerequisites
+
+- [Go 1.13+](https://go.dev/)
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker-Compose](https://docs.docker.com/compose/install/)
```

Comment:
Can we framed these as dependencies for testing? Otherwise it may appear that docker and docker compose are fundamental to the contribution.

Reply:
Well in a way they are fundamental to the contribution, to ensure that all tests are passing, or if we want to add new test cases we will need the mocks as well.